Makefile

```diff
@@ -423,7 +422,7 @@ setup-local-debugging:
 	&& . ${MAKEFILE_DIR}/devops/scripts/load_env.sh ./templates/core/.env \
```

Comment:
Why not these other ones too?

Reply:
We've considered the 2 main env files something a user would interact with and hence be outside the TRE OSS directoy. Everything else (the private one) is kind off "internal" and automated for the user which is why it got that prefix.